src/tutorial/runtime/introduction.rst

```diff
@@ -291,65 +337,165 @@ Where Do All The Files Go?
 
          ~/cylc-run/<workflow-name>/log/job/<cycle-point>/<task-name>/<job-submission-num>/
 
-      .. image:: ../img/cylc-gui-view-log.png
-         :align: center
-         :scale: 75%
+      .. TODO REPLACE THIS IF APPROPRIATE
 
+         .. image:: ../img/cylc-gui-view-log.png
+            :align: center
+            :scale: 75%
 
-Running A Workflow
-------------------
+
+Installing A Workflow
+---------------------
 
 .. ifnotslides::
 
-   It is a good idea to check a workflow for errors before running it.
-   Cylc provides a command which automatically checks for any obvious
-   configuration issues called ``cylc validate``, run via:
+   .. seealso::
 
-.. code-block:: sub
+      :ref:`The full guide to Cylc install <Installing-workflows>`.
+
+   To allow you to separate the development and running of workflows
+   Cylc provides a :term:`cylc install <install>` command.
+
+
+From ``cylc-src/my_workflow``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
 
-   cylc validate <path/to/workflow>
+   cylc install my_workflow
 
 .. ifnotslides::
 
-   Here ``<path/to/workflow>`` is the path to the workflow's location within the
-   filesystem (so if we create a workflow in ``~/cylc-run/foo`` we would put
-   ``~/cylc-run/foo/flow.cylc``).
+   will install your workflow in ``~/cylc-run/my_workflow/runN``.
 
-   Next we can run the workflow using the ``cylc play`` command.
+From a workflow developed in an arbitrary location
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. code-block:: sub
 
-   cylc play <name>
+   cylc install -C /some/location/my_other_workflow
 
 .. ifnotslides::
 
-   The ``name`` is the name of the :term:`run directory` (i.e. ``<name>``
-   would be ``foo`` in the above example).
+   will install your workflow in ``~/cylc-run/my_other_workflow/runN``.
 
-.. note::
+   .. note::
+
+      You can still develop workflows in ``~/cylc-run``, but this is not
+      recommended because development work may
+      change the behaviour of the workflow as it is running.
```

Comment:
> change the behaviour of the workflow as it is running

`cylc reinstall` suffers exactly the same problem.

Don't really need to raise this in the tutorial, if we do good to trim it down.

```suggestion
      You can edit workflows directly in ``~/cylc-run``, but this is not
      recommended.
```